model/AI/Properties/hyperparameter.md

```diff
@@ -8,7 +8,7 @@ Records a hyperparameter used to build the AI model contained in the AI package.
 
 ## Description
 
-This field records a hyperparameter value.
+Records a hyperparameter value.
 Hyperparameters are parameters of the machine learning model that are used to control the learning process,
 for example the optimization and learning rate used during the training of the model.
```

Comment:
Suggest adding, a hyperparameter is a parameter that is set before the training process begins and is used to control the behavior of the learning algorithm. Hyperparameters are not learned from the data, but are instead set by the developer or chosen through a process of trial and error.

 add in the for example, learning rate, batch size, or number of layers.

Reply:
Done